packages/client/src/offline/actions.ts

```diff
@@ -92,10 +92,10 @@ export type ApplicationConfigUpdatedAction = {
   type: typeof UPDATE_OFFLINE_CONFIG
   payload: { config: IApplicationConfig }
 }
-export const UPDATE_OFFLINE_INTEGRATIONS =
+export const UpdateOfflineIntegrationsAction =
```

Comment:
@CornelMaze There might have been a misunderstanding. I would have liked `ApplicationConfigUpdatedIntegrations` to be changed to `UpdateOfflineIntegrationsAction` and not this one.

Reply:
ok

Reply:
Alright